resumo-da-compra/README.md

```diff
@@ -0,0 +1,44 @@
+# Resumo da compra
+
+Resumo da compra é para ajudar a pessoa que fez um pedido no restaurante e deseja rever os itens que foram pedidos.
+
+As variáveis dos itens do pedido são:
+* Descrição do item
+* Quantidade de itens
+* Preço unitário
+
+Objetivo da rotina é responder em uma string o resumo da compra
+
+# Exemplos
+
+* Exemplo 1: vai um jovem no restaurante, faz o pedido de 1 refrigerante de 3 reais
+
+ no resumo deve exibir:
+ `Confirma a compra de: 1 'refrigerante' totalizando 3 reais?`
```

Comment:
Se for 1 real tem que respeitar o singular também né @ptcmariano ? Sugiro ter um exemplo pra isso tipo:

```
* Exemplo 1: vai um jovem na padaria e compra uma bala que custa 1 real

 no resumo deve exibir:
 `Confirma a compra de: 1 'bala' totalizando 1 real?`
```

Sei que ta bem complicado atualmente a gente gastar 1 real em qualquer lugar... geralmente a gente sempre gasta "reais" :joy: .... mas só perguntei pois acho que pode ser um caso válido e simples para testar :smile:

Reply:
boa discussão, eu acho que a principio não devemos se preocupar com singular/plural, como foi definido na linha 42, e daí se o propósito for resolvido vamos avançando em melhorar os resultados